ping-centre/PingCentre.jsm

```diff
@@ -336,39 +319,6 @@ class PingCentre {
     return payload;
   }
 
-  async sendPing(data, options) {
-    if (!this.enabled) {
-      return Promise.resolve();
-    }
-
-    const payload = await this._createPing(data, options);
```

Comment:
Is `_createPing` dead code now and can we delete it too?

Reply:
Or are we going to possibly use it in the upcoming patches?

Reply:
Yes, it's still useful for the upcoming patches, such as the event pings.